src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -96,6 +134,35 @@
     (.runScriptlet "require 'jar-dependencies'")
     (.runScriptlet "require 'puppet/server/master'")))
 
+(schema/defn ^:always-validate config->puppet-config :- HashMap
```

Comment:
No...  Weird.  Removed in it's own commit since this might be a pain to merge back up into master.